### PR TITLE
Delete Button, Username login, Updated Scrapeer, Security Fixes

### PR DESCRIPTION
**Pull request by AzukaChan**
I completely deleted my fork and started again, now there are 2 branches of my fork. Branding has changed back to neutral and everything appears to be working good, but if @KevinJDurant wouldn't mind checking the search torrent feature that would be handy. Searching for torrents works fine on the live demo but not on XAMPP (they are the same files) and I'm not sure whats up with that.

Edit: 
tested the search feature with @KevinJDurant 's fork on XAMPP and it didnt work so maybe its just XAMPP.

Edit 2:
Repulled my forks and uploaded to live demo; search feature is fine.. Good to pull if you like.

Branches
------------
Master: Neutral branding so you are able to pull at your convenience.
OpenTorrents: Live setup from the demo site @ http://torrents.azukachan.com

Changes
------------
1)Users can delete torrents from my-torrents.php
2)Branding is now neutral
3)Delete torrent message changed.
4)Scraper.php updated
5)Remove < code > from upload form for security reasons.
6)Users now login with username instead of email address.

### CODE REVIEW

File: en/view/my-torrents.php [hunk 1]
```diff
@@ -187,7 +187,7 @@
                             <td data-label="Leech">'.$row["leechers"].'</td>
                             <td data-label="Download">
                                 <a href="'.$row["magnet"].'"><span class="glyphicon glyphicon-magnet link"></span></a>
-                                <a href="delete_torrent_file.php?delete_id='.$row["torrents_id"].'"  class="delete" data-confirm="Are you sure to delete this torrent file?"><span class="glyphicon glyphicon-trash link"></span></a>
+                                <a href="delete_torrent_file.php?delete_id='.$row["torrents_id"].'"  class="delete" data-confirm="Are you sure you want to permanently remove this torrent from the databse?"><span class="glyphicon glyphicon-trash link"></span></a>
```
Comment (KevinJDurant) [inline en/view/my-torrents.php]:
```suggestion
                                <a href="delete_torrent_file.php?delete_id='.$row["torrents_id"].'"  class="delete" data-confirm="Are you sure you want to permanently remove this torrent from the database?"><span class="glyphicon glyphicon-trash link"></span></a>
```

File: en/account/login.php [hunk 2]
```diff
@@ -1,188 +1,188 @@
...
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Standard Meta -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="OpenTorrents: Free Software, Games and Music!">
```
Comment (KevinJDurant) [inline en/account/login.php]:
```suggestion
    <meta name="description" content="OpenTorrentSite: Free Software, Games and Music!">
```

File: en/account/login.php [hunk 3]
```diff
@@ -1,188 +1,188 @@
...
+    <meta name="author" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Login | OpenTorrents</title>
```
Comment (KevinJDurant) [inline en/account/login.php]:
```suggestion
    <title>Login | OpenTorrentSite</title>
```

File: en/account/login.php [hunk 4]
```diff
@@ -1,188 +1,188 @@
...
+        <div class="container">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="../../index.php">OpenTorrents</a>
```
Comment (KevinJDurant) [inline en/account/login.php]:
```suggestion
                <a class="navbar-brand" href="../../index.php">OpenTorrentSite</a>
```

File: en/account/login.php [hunk 5]
```diff
@@ -1,188 +1,188 @@
...
+        </div>
+        <!-- /.row -->
+
+        <hr>
+
+        <!-- Footer -->
+        <footer>
+            <div class="row">
+                <div class="col-lg-12">
+                    <p>Happy Downloading ~</p>
```
Comment (KevinJDurant) [inline en/account/login.php]:
```suggestion
                    <p>Copyright &copy; Your Website 2017</p>
```

**Comment by KevinJDurant** [comment]
@AzukaChan I have reviewed the code and suggested some changes. I can't commit my review to your branch, I'll merge once these small issues have been fixed.

**Comment by AzukaChan** [comment]
@KevinJDurant 
My bad for the small issues.
Suggestions have been added.

**Comment by KevinJDurant** [comment]
@AzukaChan I have squashed your changes into 1 commit and merged them into the master branch. Thanks for your contributions!

**Pull request merged**